src/adjustItems.js

```diff
@@ -0,0 +1,28 @@
+function addTodo(arr) {
+  const textGetter = document.getElementById('TodoDesc');
+  arr.push({
+    description: textGetter.value,
+    completed: false,
+    index: arr.length,
+  });
+  textGetter.value = '';
+}
+function deleteTodo(arr, index) {
+  const cont = document.getElementById(index);
+  cont.remove();
+  arr.splice(index - 1, 1);
+  for (let i = 0; i < arr.length; i += 1) {
+    arr[i].index = i + 1;
+  }
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
+}
+
+function deleteMarked(arr) {
+  arr = arr.filter((item) => item.completed === false);
+  arr.forEach((item) => item.add());
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
+  return arr;
+}
```

Comment:
:heavy_check_mark: great work

Reply:
The edit function doesn't accept ENTER as submit, but a click that is outside of the text area. I can change it if it's a requirement but if it's OK as is, I wouldn't want to risk breaking something new as this was my second review request.